code/modules/cm_marines/overwatch.dm

```diff
@@ -798,7 +798,9 @@
 	log_attack("[key_name(user)] fired an orbital bombardment in [A.name] for squad '[current_squad]'")
 
 	/// Project ARES interface log.
-	GLOB.ares_link.log_ares_bombardment(user.name, ob_name, "X[x_bomb], Y[y_bomb] in [A.name]")
+	var/datum/ares_datacore/datacore = GLOB.ares_datacore
+	if(ares_can_log())
```

Comment:
This should really be like inside the proc you are calling if it's required before every time you call the proc